tests/test_osmnx.py

```diff
@@ -29,6 +29,7 @@
 from typeguard import TypeCheckError
 
 import osmnx as ox
+from osmnx.bearing import _extract_edge_bearings
```

Comment:
Remove this import and access this function via `ox.bearing._extract_edge_bearings` below when you need it.